component/org.wso2.carbon.identity.dpop/src/main/java/org/wso2/carbon/identity/dpop/validators/DPoPHeaderValidator.java

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
Add license header

Reply:
License header is available in the class